subiquitycore/pubsub.py

```diff
@@ -14,9 +14,17 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import asyncio
+import enum
 import inspect
 
 
+class MessageChannels(enum.IntEnum):
```

Comment:
This should be in the subiquity package somewhere I think.

Reply:
That will make things a little weird for the pubsub broadcast in subiquitycore/controllers/network.py.
Otherwise I agree.